front/src/modules/rollingStock/components/RollingStockEditor/RollingStockEditorFormModal.tsx

```diff
@@ -52,7 +59,7 @@ const RollingStockEditorFormModal = ({
       {!errorObject ? (
         <span className="text-primary mb-3">{mainText}</span>
       ) : (
-        displayErrorObject(errorObject)
+        displayErrorObject(errorObject as RollingStockIsUsed)
```

Comment:
Remove the cast here
```suggestion
        displayErrorObject(errorObject)
```